src/modules/mc_rate_control/mc_rate_control_params.c

```diff
@@ -292,3 +292,17 @@ PARAM_DEFINE_FLOAT(MC_YAWRATE_K, 1.0f);
  * @group Multicopter Rate Control
  */
 PARAM_DEFINE_INT32(MC_BAT_SCALE_EN, 0);
+
+/**
+ * First order low pass filter cutoff frequency for yaw rate control
+ *
+ * Reduces vibrations by lowering high frequency torque caused by rotor acceleration.
+ * 0 disables the filter
+ *
+ * @min 0
+ * @max 10
+ * @unit Hz
+ * @decimal 3
+ * @group Multicopter Rate Control
+ */
+PARAM_DEFINE_FLOAT(MC_YAWRATE_FC, 2.f);
```

Comment:
I'm not sure if I have a better suggestion, but does this naming make sense?

Something more along the lines of yaw torque filter cutoff or in someway distinguish that it's specifically the output?

Reply:
The suggested name from https://github.com/PX4/PX4-Autopilot/pull/21434 was `MC_YAW_CUTOFF`. I like the `_CUTOFF` prefix but `MC_YAW` is too inaccurate. Also, it would be problematic if we want to add a yaw filter (not yawrate like now).
What about `MC_YAW_TQ_CUTOFF` or `MC_Y_TRQ_CUTOFF` ?

Reply:
Changed it to `MC_YAW_TQ_CUTOFF` in [28bc950](https://github.com/PX4/PX4-Autopilot/pull/24173/commits/28bc95033a6ab3e0f5c6f5e21a4bcecbb6241a19)